skll/data/readers.py

```diff
@@ -1052,8 +1077,9 @@ def _sub_read(self, file):
 
 def safe_float(text, replace_dict=None, logger=None):
     """
-    Attempts to convert a string to an int, and then a float, but if neither is
-    possible, returns the original string value.
+    Convert string to int and then float.
```

Comment:
```suggestion
    Convert string to int.
    If not possible,  then float.
```

Reply:
That's not actually correct. It does do both.

Reply:
Good catch. I'll tweak it to be more accurate.